src/com/content-event/event-idea.js

```diff
@@ -30,35 +29,34 @@ export default class ContentEventIdea extends Component {
 	componentDidMount() {
 		$ThemeIdea.GetMy([this.props.node.id])
 		.then(r => {
+			//console.log(r);
 			if ( r.ideas ) {
-				this.setState({"ideas": r.ideas});
+				this.setState({'ideas': r.ideas, 'enableSubmit': canHaveMoreIdeas(r.ideas)});
 			}
 			else {
-				this.setState({"ideas": {}});
+				this.setState({'ideas': {}});
 			}
 		})
 		.catch(err => {
-			this.setState({"error": err});
+			this.setState({'error': 'Error fetching your previous suggestions. Make sure you are logged in.'});
```

Comment:
Should use `"` around `"Error fetching your previous suggestions..."` and not `'`.